### PR TITLE
Added the 'All-Contributors' package, added applicable scripts to pac…

### PR DESCRIPTION
**Pull request by corruptedzulu**
…kage.json, added myself as first listed contributor (manual reordering might be useful). Also managed to misspell the word "emoji" on the branch name...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 infamous
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors)
```
Comment (trusktr) [inline README.md]: Maybe just "contributors". WDYT?

Comment (corruptedzulu) [reply README.md]: I personally don't have an issue with dropping the "all", though that part was provided by the cli tool. I'm not sure if that's just from an init step or if it replaces it on each update. If the latter, we'd have to work around that in some way (e.g. contribute to their project, manually change it, write a script to correct it after running the cli tool).

Comment (trusktr) [reply README.md]: If the latter, we can just leave it like that then, no biggie. I'm guessing it was probably on init. Yeah, forking it and using our forked version of the CLI tool would work too, but maybe too much work.

**Comment by trusktr** [comment]
LGTM, let's close out the related issue after merging too.

**Pull request merged**